### PR TITLE
Cleanup: refactor dev guides

### PR DESCRIPTION
**Pull request by rhoads-zach**
@inoxx03 mind taking a quick look?

### CODE REVIEW

File: docs/topics/health-check-mission-basic-interaction-wf-swarm.adoc [hunk 1]
```diff
@@ -8,19 +8,19 @@ In the following we demonstrate the basic interactions on the command line that
 +
 [source,bash,options="nowrap",subs="attributes"]
 ----
-$ curl http://{app-name}-{project-name}.192.168.64.3.nip.io/api/greeting
+$ curl http://{app-name}-{project-name}.1ab5.starter-us-east-1.openshiftapps.com/api/greeting
 {"content":"Hello, World!"}
 ----
 
 . To simulate a service internal failure and to trigger the openshift selfhealing capabilities, invoke the `/api/killme` endpoint and verify the availability of the `/api/greeting` endpoint shortly after that. It should return a HTTP status 503:
```
Comment (inoxx03) [inline docs/topics/health-check-mission-basic-interaction-wf-swarm.adoc]: I think OpenShift should be written in camel case. WDYT?

openshift > OpenShift

File: docs/topics/configmap-mission-deploy-booster.adoc [hunk 2]
```diff
@@ -80,7 +78,7 @@ data:
 ----
 endif::configmap-wf-swarm[]
 
-. Use maven to start the deployment to your {OpenShiftLocal}.
+. Use maven to start the deployment to your {OpenShiftOnline}.
```
Comment (inoxx03) [inline docs/topics/configmap-mission-deploy-booster.adoc]: Should we capitalize maven as `Maven`?

**Comment by Ladicek** [comment]
Agreed with both points re capitalization.

**Comment by rhoads-zach** [comment]
Fixed typos. Merging.

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/e4ac040b1fb6beab5e6d60e8d98ff247f711c185

**Pull request closed without merging**